kumascript/macros/js_property_attributes.ejs

```diff
@@ -47,12 +47,12 @@ const text = mdn.localStringMap({
     no: "不可"
   },
   "ko": {
-    header: `Property attributes of <code>${env.title}</code>`,
-    writableName: "Writable",
-    enumerableName: "Enumerable",
-    configurableName: "Configurable",
-    yes: "yes",
-    no: "no"
+    header = 'Property attributes of <code>' + env.title + '</code>';
+    writable_name = '쓰기 가능';
+    enumerable_name = '열거 가능';
+    configurable_name = '설정 가능';
+    yes = '가능';
+    no = '불가능';
```

Comment:
These semicolons should be commas (and cause `SyntaxError: Unexpected token ';' while compiling ejs`):

```suggestion
    header = 'Property attributes of <code>' + env.title + '</code>',
    writable_name = '쓰기 가능',
    enumerable_name = '열거 가능',
    configurable_name = '설정 가능',
    yes = '가능',
    no = '불가능',
```